ioc_copier/ioc_copier.py

```diff
@@ -55,6 +57,10 @@ def replace_line(ioc, line):
     return:
         Tne new line of text.
     """
+    global asub_record
+    if "aSubRecord" in line and not asub_record:
```

Comment:
Might be better to check for `record(aSub` rather than `aSubRecord`, this will flag up where it's used in DBs